appcenter-link-scripts/src/android/index.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
         }
         return inquirer.prompt([{
             type: 'input',
-            message: 'What is the Android App Secret?',
+            message: 'What secret does your Android app use? [None]',
```

Comment:
@pniko Question, so user has to delete `[None]` and then enter the app secret in the prompt?

Reply:
He is just changing the message, there is still no default value.

Reply:
It's just to signal you can hit return. We are updating the Getting Started pages to stop directing users to flip to another app for the other secret. Narrowing the Getting Started instructions to the platform they are staring at makes things a bit cleaner until we let users group apps together in the future.